src/components/tutor-input.cjsx

```diff
@@ -83,7 +83,11 @@ TutorDateInput = React.createClass
     wrapperClasses = ["form-control-wrapper", "tutor-input"]
 
     now = TimeStore.getNow()
-    value = if @props.value then new moment(@props.value) else null
+    value = @props.value
+    value = if value and value.getTime and not isNaN(value.getTime())
```

Comment:
This code is more for prevention, I think setting the max and required props worked helped to eliminate the error state you experienced earlier.